packages/swingset-theme-hashicorp/src/components/code-block/code-block.tsx

```diff
@@ -0,0 +1,43 @@
+'use client' //Library calls hooks internally
+import { Highlight, themes } from 'prism-react-renderer'
+import { parseCode, parseLanguage } from './helpers'
+import { Language, MDXPreClass } from '@/types'
+import { CopyButton } from './copy-button'
+
+type CodeBlockProps = {
+  filePath: `${string}.${Language}`
+  children: React.Component
```

Comment:
```suggestion
  children: React.ReactNode
```

This is a subtle change, but the most reliable type to use for `children` is `ReactNode`.